vscode-lean4/src/leanclient.ts

```diff
@@ -346,7 +348,7 @@ export class LeanClient implements Disposable {
             protDiag.fullRange = c2p.asRange(diag.fullRange)
             return protDiag
         }
-        c2p.asDiagnostics = (diags) => diags.map(d => c2p.asDiagnostic(d))
+        c2p.asDiagnostics = async (diags) => diags.map(d => c2p.asDiagnostic(d))
```

Comment:
this is probably due to the vscode version change, please revert this since Gabriel wants this to be done in a separate PR.